en/integrations/plugin-packs/procedures/operatingsystems-as400-connector.md

```diff
@@ -0,0 +1,258 @@
+| status      | Status of the remote command (failed or not) |
+
+<!--Disks-->
+
+* Global
+
+| Metric name                      | Unit  |
+|:---------------------------------|:------|
+| disks.total.count                | count |
+| disks.active.count               | count |
+| disks.errors.count               | count |
+| disks.gap.repartition.percentage | %     |
+
+* Per *disks*
+
+| Metric name                 | Unit |
+|:----------------------------|:-----|
+| status                      |      |
+| disk.space.usage.bytes      | B    |
+| disk.space.free.bytes       | B    |
+| disk.space.usage.percentage | %    |
+| disk.space.reserved.bytes   | B    |
+
+<!--Job-Queues-->
+
+* Global
+
+| Metric name           | Unit  |
+|:----------------------|:------|
+| jobqueues.total.count | count |
+
+* Per *jobq*
+
+| Metric name                   | Unit  |
+|:------------------------------|:------|
+| status                        |       |
+| jobqueue.jobs.active.count    | count |
+| jobqueue.jobs.scheduled.count | count |
+| jobqueue.jobs.held.count      | count |
+
+<!--Jobs-->
+
+| Metric name      | Unit  |
+|:-----------------|:------|
+| jobs.total.count | count |
+
+<!--Message-Queue-->
+
+| Metric name       | Unit  |
+|:------------------|:------|
+| mq.messages.count | count |
+
+<!--Page-Faults-->
+
+| Metric name                             | Unit |
+|:----------------------------------------|:-----|
+| pagefaults.database.ratio.percentage    | %    |
+| pagefaults.nondatabase.ratio.percentage | %    |
+
+<!--SubSystem-->
+
+* Global
+
+| Metric name            | Unit  |
+|:-----------------------|:------|
+| subsystems.total.count | count |
+
+* Per *subsys*
+
+| Metric name                 | Unit  |
+|:----------------------------|:------|
+| status                      |       |
+| subsystem.jobs.active.count | count |
+
+<!--System-->
+
+* *CPU*
+
+| Metric name                                | Unit |
+|:-------------------------------------------|:-----|
+| system.processing.units.usage.percentage   | %    |
+| system.storage.pool.space.usage.percentage | %    |
+
+* *jobs*
+
+| Metric name              | Description | Unit  |
+|:-------------------------|:------------|:------|
+| system.jobs.total.count  | total: %s   | count |
+| system.jobs.active.count | active: %s  | count |
+
+* *batch_jobs*
+
+| Metric name                     | Unit  |
+|:--------------------------------|:------|
+| system.batch_jobs.running.count | count |
+| system.batch_jobs.waiting.count | count |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+This plugin works in a slightly different way than the common ones. It requires 
+a connector to communicate with the AS400/iSeries system. 
+
+You can install the connector using this command: 
+
+```shell
+yum install centreon-plugin-Operatingsystems-AS400-daemon
+```
+
+A connector can act as a boilerplate between several Hosts and several AS400 systems. 
```

Comment:
Are you sure about the word boilerplate here?

Reply:
I am